tests/common.py

```diff
@@ -827,6 +827,8 @@ def run_process(self, cmd, check=True, **args):
       return shared.run_process(cmd, check=check, **args)
     except subprocess.CalledProcessError as e:
       if check and e.returncode != 0:
+        print(e.stdout)
+        print(e.stderr)
```

Comment:
is this an intentional change?

Reply:
Yes without this tests that capture their output, but fail will hide all output, making debugging very hard.  Its unrelated but very useful :)